test/verify/check-accounts

```diff
@@ -24,6 +24,7 @@ from testlib import *
 
 
 @nondestructive
+@skipImage("Do not test cockpit package", "rhel-8-3-distropkg")
```

Comment:
How about "Do not test BaseOS packages"? "cockpit package" is a bit confusing I think, as everything here is about cockpit.